tests/test_jobs.py

```diff
@@ -0,0 +1,82 @@
+import polling2
+import pytest
+
+from domino import Domino
+from requests.exceptions import RequestException
+
+
+@pytest.fixture
+def dummy_hostname():
+    return "http://domino.somefakecompany.com"
+
+
+@pytest.fixture
+def mock_api_call_chain(requests_mock, dummy_hostname):
```

Comment:
Should this fixture have a more specific name?
```suggestion
def mock_job_start_blocking_setup(requests_mock, dummy_hostname):
```
Note if you change this here, you must elsewhere.

Reply:
Thanks, I'll just do it all in one commit.